### ISSUE TITLE
Migration to v2 of config files doesn't honour update schedule

### ISSUE THREAD

**Issue by gnarea** [dependabot/feedback#972]
See https://github.com/relaycorp/relaynet-pong/pull/85 for example. That repo uses live updates but the PR would change it to daily.

**Comment by feelepxyz** [comment]
@gnarea yeah we currently move `live` to `daily` when migrating because we don't support live schedules in the GitHub native version. We would like to support this eventually but will take some time to get this right.
